packages/internal/package.json

```diff
@@ -43,6 +43,7 @@
     "@graphql-codegen/typescript-resolvers": "2.7.4",
```

Comment:
Do we need to remove findup-sync for example from here? Need to make sure unnecessary dependencies aren't included. 

https://www.npmjs.com/package/check-unused-dependencies

Might be useful

Reply:
![image](https://user-images.githubusercontent.com/30793/203403885-f0f1112a-fb23-4d37-9a41-41bffc64b00b.png)

Good call! I'll clean it up

Reply:
I kept these (at least for now). Doesn't sound like packages that would be unneeded because I moved the paths related functions

![image](https://user-images.githubusercontent.com/30793/203405046-f12df7fc-5c9a-4653-9b55-e90cc1beca3f.png)